slice-test/slice-mapper-test/src/test/groovy/com/cognifide/slice/mapper/annotation/InheritanceTest.groovy

```diff
@@ -0,0 +1,26 @@
+package com.cognifide.slice.mapper.annotation
+
+import com.cognifide.slice.test.setup.BaseSetup
+import org.junit.Assert
+
+/**
+ * @author Mariusz Kubiś Date: 15.04.15
+ */
+class InheritanceTest extends BaseSetup {
```

Comment:
Please add following test cases:
* various mapping strategies are used (both on parent and child classes, all combinations should be covered)
* @SliceResource applied on parent and (not) on child class